### PR TITLE
adding ingress perf first commit

### PR DESCRIPTION
**Pull request by paigerube14**
First run with no jenkinsfile errors: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/ingress-perf/2/console

### CODE REVIEW

**Comment by paigerube14** [comment]
@qiliRedHat how long does the test normally run? It was taking over 4 hours so I aborted it

**Comment by paigerube14** [comment]
> @qiliRedHat how long does the test normally run? It was taking over 4 hours so I aborted it

Was using the router-perf folder still. Updated now and running

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/ingress-perf/5/console

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        )
+        string(
+            name: 'TOLERANCY',
+            defaultValue: '-10',
+            description: '''
+                Tolerancy to replace value of tolerancy in <a href="https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/ingress-perf">tolerancy</a>.<br>
+                i.e a 10 would mean any metric 10% higher than the baseline metric will be considered an error<br>
+                , and -10 would mean the opposite, any metric at least 10% below the baseline value will be considered an error.<br>
+                Set to blank will not use TOLERANCY_RULES.
+            '''
```
Comment (qiliRedHat) [inline Jenkinsfile:116]:
```suggestion
            defaultValue: '20',
            description: '''Defines a regression tolerancy percent. By default 20'''
```

**Comment by paigerube14** [comment]
Yes will have to add. It has to run one time for the index to be found. Will add today.

**Comment by paigerube14** [comment]
@qiliRedHat the data source is properly added now and can be seen in the grafana ingress dashboard

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+                            export ES_SERVER_BASELINE="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
+                            mkdir -p ~/.kube
+                            cp $WORKSPACE/flexy-artifacts/workdir/install-dir/auth/kubeconfig ~/.kube/config
+                            oc config view
+                            oc projects
+                            ls -ls ~/.kube/
+                            env
+                            cd workloads/ingress-perf
+                            ./run.sh |& tee "ingress_perf.out"
+                            ! grep "Benchmark comparison failed" ingress_perf.out
```
Comment (qiliRedHat) [inline Jenkinsfile:217]:
```suggestion
                            ! egrep -i "lower than baseline|higher than baseline|error" ingress_perf.out
```

**Comment by qiliRedHat** [comment]
Updated the datasource name to have big I in 'Ingress', now I can see our test results on https://grafana.rdu2.scalelab.redhat.com:3000/d/nlAhmRyVk/ingress-perf?orgId=1&var-datasource=QE%20Ingress-perf&var-platform=AWS&var-uuid=a682b97e-53ff-44c2-a4bd-8bc4a320a290&var-termination=edge&var-termination=http&var-termination=passthrough&var-termination=reencrypt&var-latency_metric=avg_lat_us&var-compare_by=uuid.keyword&var-concurrency=18&from=now-2w&to=now

**Comment by paigerube14** [comment]
@qiliRedHat is this good to merge and we can add functionality in the future

**Pull request merged**